server/src/server.rs

```diff
@@ -0,0 +1,13 @@
+use actix_web::{App, HttpServer};
+use crate::controllers::{test_controller};
+
+pub fn start() {
+    HttpServer::new(|| {
+        App::new()
+            .configure(test_controller::routes)
+    })
+        .bind("0.0.0.0:5000")
```

Comment:
Could we find a way to make this configurable? So that we avoid hardcoded values.